src/ralph/security/models.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.assets.models.base import BaseObject
+from ralph.lib.mixins.models import TaggableMixin, TimeStampMixin
+from ralph.lib.permissions import PermByFieldMixin
+
+
+from ralph.security.choices import Risk, ScanStatus
+
+
+class Vulnerability(
+    PermByFieldMixin,
+    TimeStampMixin,
+    TaggableMixin,
+    models.Model,
+):
+    name = models.CharField(
+        verbose_name=_("name"),
+        max_length=1024,
+        unique=False)
+    patch_deadline = models.DateTimeField(null=True, blank=True)
+    risk = models.PositiveIntegerField(choices=Risk(), null=True, blank=True)
+    external_vulnerability_id = models.IntegerField(
+        unique=True,  # id means id
+        null=True,
+        blank=True,
+        help_text=_('Id of vulnerability from external system'),
+    )
+
+    @property
+    def is_deadline_exceeded(self):
+        result = True if self.patch_deadline < datetime.now() else False
+        return result
+
+    @property
```

Comment:
there is property (or method?) created automatically for that, probably sth like `risk_display_name` or `get_risk_display_name`